.github/labeler.yml

```
@@ -118,5 +135,12 @@
 "Mods: No Hope":
   - data/mods/No_Hope/**/*
 
+"Mods: TropiCataclysm 🌴":
+  - data/mods/TropiCataclysm/**/*
+
 "Mods":
   - data/mods/**/*
+
+"Weather":
+  - "src/weather*.*"
+  - data/json/weather_type.json
```

```suggestion
"Mechanics: Weather":
  - "**/weather**"
```
We cover these files
> fd --type f | rg ".+/weather"
./data/core/weather.json
./data/mods/aftershock_exoplanet/weather_type.json
./data/json/weather_type.json
./data/json/ui/weather.json
./src/weather_gen.cpp
./src/weather.cpp
./src/weather.h
./src/weather_gen.h
./lang/string_extractor/parsers/weather_type.py
./tests/weather_test.cpp
./src/weather_type.cpp
./src/weather_type.h

---

I thought we did and then I couldn't find them.  I feel ridiculous since it's a small file. lol